src/components/FindMovie/FindMovie.js

```diff
@@ -1,55 +1,104 @@
-import React from 'react';
+import React, { useState } from 'react';
+import classNames from 'classnames';
 import './FindMovie.scss';
+import PropTypes from 'prop-types';
 
 import { MovieCard } from '../MovieCard';
-import movies from '../../api/movies.json';
-
-export const FindMovie = () => (
-  <>
-    <form className="find-movie">
-      <div className="field">
-        <label className="label" htmlFor="movie-title">
-          Movie title
-        </label>
-
-        <div className="control">
-          <input
-            type="text"
-            id="movie-title"
-            placeholder="Enter a title to search"
-            className="input is-danger"
-          />
+import { getMovie } from '../../api/api';
+
+export const FindMovie = ({ addMovie }) => {
+  const [movie, setMovie] = useState([]);
+  const [title, setTitle] = useState('');
+  const [error, setError] = useState(false);
+
+  const movieSearch = (name) => {
+    getMovie(name)
+      .then((data) => {
+        setMovie({
+          title: data.Title,
+          description: data.Plot,
+          imgUrl: data.Poster,
+          imdbUrl: `https://www.imdb.com/title/${data.imdbID}/`,
+          imdbId: data.imdbID,
+        });
+        setError(data.Error);
+      });
+  };
+
+  return (
+    <>
+      <form
+        className="find-movie"
+        onSubmit={(event) => {
+          event.preventDefault();
+
+          if (!error && movie.length !== 0) {
+            addMovie(movie);
+
+            setMovie([]);
+            setTitle('');
+          }
+        }}
+      >
+        <div className="field">
+          <label className="label" htmlFor="movie-title">
+            Movie title
+          </label>
+
+          <div className="control">
+            <input
+              type="text"
+              id="movie-title"
+              placeholder="Enter a title to search"
+              className={classNames('input', { 'is-danger': error })}
+              value={title}
+              onChange={(event) => {
+                setTitle(event.target.value);
+                setError(false);
+                setMovie([]);
+              }}
+            />
+          </div>
+
+          {error && (
+            <p className="help is-danger">
+              Can&apos;t find a movie with such a title
+            </p>
+          )}
         </div>
 
-        <p className="help is-danger">
-          Can&apos;t find a movie with such a title
-        </p>
-      </div>
-
-      <div className="field is-grouped">
-        <div className="control">
-          <button
-            type="button"
-            className="button is-light"
-          >
-            Find a movie
-          </button>
+        <div className="field is-grouped">
+          <div className="control">
+            <button
+              type="button"
```

Comment:
if you'll change the type of your button to `submit` you give users opportunity to find a movie with the `enter` key